src/fakeData/projects.ts

```diff
@@ -0,0 +1,103 @@
+import { Project, ProjectStatus } from 'utils/types';
+
+export const PROJECTS: Project[] = [
+   {
+      id: 1,
+      name: 'Project 1',
+      status: ProjectStatus.Pending,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [1, 20, 18, 15],
+      pmId: 1,
+   },
+   {
+      id: 2,
+      name: 'Project 2',
+      status: ProjectStatus.Completed,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [2, 22, 19, 28],
+      pmId: 2,
+   },
+   {
+      id: 3,
+      name: 'Project 3',
+      status: ProjectStatus.Completed,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [3, 28, 17, 26],
+      pmId: 3,
+   },
+   {
+      id: 4,
+      name: 'Project 4',
+      status: ProjectStatus.Completed,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [4, 15, 29, 14],
+      pmId: 4,
+   },
+   {
+      id: 5,
+      name: 'Project 5',
+      status: ProjectStatus.Completed,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [5, 19, 25, 20],
+      pmId: 5,
+   },
+   {
+      id: 6,
+      name: 'Project 6',
+      status: ProjectStatus.Pending,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [6, 31, 21, 26],
+      pmId: 6,
+   },
+   {
+      id: 7,
+      name: 'Project 7',
+      status: ProjectStatus.Completed,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [7, 19, 20, 16],
+      pmId: 7,
+   },
+   {
+      id: 8,
+      name: 'Project 8',
+      status: ProjectStatus.Pending,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [8, 25, 17, 21],
+      pmId: 8,
+   },
+   {
+      id: 9,
+      name: 'Project 9',
+      status: ProjectStatus.InProgress,
+      client: 'ZinZa. Co',
+      description: 'Lorem Ipsum is simply dummy text',
+      startDate: '2021-09-27',
+      endDate: '2021-10-27',
+      memberId: [9, 30, 14, 16],
+      pmId: 9,
+   },
+];
```

Comment:
```suggestion
.map<Project>((project) => {
   return {
      ...project,
      pm: USERS.find(user => user.id === project.pmId),
      members: USERS.filter(user => project.memberId.includes(user.id)),
   }
});
```

A nghĩ nên để type Project có thuộc tính là members và pm như trước, như vậy sau này sẽ dễ ghép api hơn (tương tự với task)